docs/sources/example-rules/github/github.yaml

```diff
@@ -355,92 +414,35 @@ endpoints:
         encoding: "JSON"
       - !<redact>
         jsonPaths:
-          - "$..owner.avatar_url"
-          - "$..owner.gravatar_id"
-          - "$..owner.url"
-          - "$..owner.html_url"
-          - "$..owner.followers_url"
-          - "$..owner.following_url"
-          - "$..owner.gists_url"
-          - "$..owner.starred_url"
-          - "$..owner.subscriptions_url"
-          - "$..owner.organizations_url"
-          - "$..owner.repos_url"
-          - "$..owner.events_url"
-          - "$..owner.received_events_url"
-      - !<pseudonymize>
-        jsonPaths:
-          - "$..owner.login"
-          - "$..owner.id"
-          - "$..owner.node_id"
-          - "$..owner.email"
-        encoding: "JSON"
-      - !<redact>
-        jsonPaths:
-          - "$..user.avatar_url"
-          - "$..user.gravatar_id"
-          - "$..user.url"
-          - "$..user.html_url"
-          - "$..user.followers_url"
-          - "$..user.following_url"
-          - "$..user.gists_url"
-          - "$..user.starred_url"
-          - "$..user.subscriptions_url"
-          - "$..user.organizations_url"
-          - "$..user.repos_url"
-          - "$..user.events_url"
-          - "$..user.received_events_url"
-      - !<pseudonymize>
-        jsonPaths:
-          - "$..user.login"
-          - "$..user.id"
-          - "$..user.node_id"
-          - "$..user.email"
-        encoding: "JSON"
-      - !<redact>
+          - "$..['requested_reviewers[*]','owner','user','closed_by','actor','assignee','creator','merged_by'].avatar_url"
```

Comment:
actually, I think it's clearer to do the reverse

`"$..owner.['avatar_url','url','html_url','followers_url', ...]"`

but maybe need both?

`"$..['owner','user',...].['avatar_url','url','html_url','followers_url', ...]"`

Reply:
Yeah thanks, now YAML is 850 lines (from 1300; originally before this PR > 2k lines). I have to do a dirty quick because JSONPath expression should be different if there are based on a object or plain properties; probably we could find a better way on next iteration.